src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.searching {
+  width: 100%;
+  margin-top: 20px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  position: absolute;
+}
+
+.searching:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searching:focus {
+  outline: none;
+  border-radius: 3px;
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.bigSearch {
+  height: 70px;
+  top: -8px;
```

Comment:
don't use `top`

Reply:
don't use at all or change on margin-top? 
if don't use it, I have this problem:
![image](https://user-images.githubusercontent.com/125828769/220326198-2354256e-c1b1-485c-aec2-c8474fb74ca9.png)